starport/pkg/protoanalysis/import.go

```diff
@@ -0,0 +1,44 @@
+package protoanalysis
+
+import (
+	"fmt"
+
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+)
+
+// IsProtoImported returns true if the proto file is imported in the provided proto file
+func IsProtoImported(path, protoImport string) (bool, error) {
+	f, err := ParseFile(path)
+	if err != nil {
+		return false, err
+	}
+
+	protoImport = fmt.Sprintf("%s.proto", protoImport)
+	for _, dep := range f.Dependencies {
+		if dep == protoImport {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+// EnsureProtoImported add the proto file import in the proto file content in case it's not defined
+func EnsureProtoImported(content, protoImport, protoPath, importPlaceholder string, replacer placeholder.Replacer) string {
```

Comment:
I don't think we should make protoanalysis package be aware of placeholder.Replacer. because protoanalysis is a generic package.